extensions/wikia/ContentReview/models/ReviewModel.php

```diff
@@ -233,11 +233,26 @@ public function getReviewedContent( $wiki_id, $page_id, $status ) {
 			->WHERE( 'wiki_id' )->EQUAL_TO( $wiki_id )
 			->AND_( 'page_id' )->EQUAL_TO( $page_id )
 			->AND_( 'status' )->EQUAL_TO( $status )
-			->LIMIT(1)
```

Comment:
It's not necessary since `wiki_id`, `page_id` and `status` form a unique key in our DB.